packages/components/src/List/List.component.js

```diff
@@ -54,15 +54,20 @@ ListToolbar.propTypes = {
 	renderers: PropTypes.object,
 };
 
-function DisplayModeComponent({ displayMode, id, list, useContent, virtualized, t }) {
+function DisplayModeComponent({ displayMode, id, list, rowHeight, useContent, virtualized, t }) {
 	const translatedList = Object.assign({}, list, { t });
 	if (useContent) {
 		return <Content id={id && `${id}-content`} displayMode={displayMode} {...translatedList} />;
 	}
 	if (virtualized) {
 		return (
 			<div className={'tc-list-display-virtualized'}>
-				<ListToVirtualizedList id={id} displayMode={displayMode} {...translatedList} />
+				<ListToVirtualizedList
+					id={id}
+					displayMode={displayMode}
+					rowHeight={rowHeight}
```

Comment:
Can you add or modify the tests to ensure that the info is passed ?

Reply:
added test on ListToVirtualizedList component